src/util/errors.ts

```diff
@@ -1,5 +1,18 @@
 import { ValidationError } from 'class-validator';
 
+export enum HttpResponseCode {
+	OK = 200,
```

Comment:
Can we rename this to `Ok` instead? Sorry, it's a bit pedantic

Reply:
Sure that's fine. Are the rest OK (see what I did there)?